kustomize/mariadb-cluster/base/mariadb-galera.yaml

```diff
@@ -123,6 +123,8 @@ spec:
     max_connections=10240
     open_files_limit=10240
     max-connect-errors=1000000
+    wsrep_retry_autocommit=0
```

Comment:
Will openstack keep track of failed commits and retry them? How many times?

Reply:
Yes. Openstack does do retries. That is why we need to disable it in mariadb